cldf/parameters.csv

```diff
@@ -48,7 +48,7 @@ ID,Name,Concepticon_ID,Concepticon_Gloss
 47_house,house,1252,HOUSE
 48_in,in,1460,IN
 49_knee,knee,1371,KNEE
-50_toknow,to know,1410,KNOW (SOMETHING)
+50_toknow,to know,3626,KNOW
```

Comment:
Who changed this concepticon mapping here? We could have just asked Tim, so we don't need to go for a less specific concept. This is just emphasizing that we need to stop post-editing the concept lists in huge bulk edits, as so many details, where we know the people and can ask them, just get lost here.

Reply:
Just raised an issue, ignore for this PR: https://github.com/concepticon/concepticon-data/issues/820